Engine/Module/ModuleInput.cpp

```diff
@@ -195,9 +202,12 @@ update_status ModuleInput::PreUpdate()
 			break;
 
 		case SDL_DROPFILE:
+			/* TODO: This
 			char* dropped_filedir = event.drop.file;
 			App->editor->project_explorer->CopyFileToSelectedFolder(dropped_filedir);
 			SDL_free(dropped_filedir);
+			*/
```

Comment:
Why commenting this?

Reply:
Because incoming path is in windows style, and it should be transformed to physfs one in order to access it with our classes. I'm not really sure how to do this atm.